resources/metadata/tmsExport.yaml

```diff
@@ -14,15 +14,33 @@ spec:
   inputs:
     secrets:
       - name: credentialsId
-        description: Jenkins 'Secret text' credentials ID containing service key for SAP Cloud Transport Management service.
+        description: Jenkins 'Secret text' credentials ID containing service key for SAP Cloud Transport Management (CTMS) service or SAP Cloud Application Lifecycle Management (CALM) service.
```

Comment:
Change tmsServiceKey to serviceKey in line 12.

Reply:
done